packages/email-templates/src/templates/certificationKeyAirdropped.ts

```diff
@@ -0,0 +1,22 @@
+import handlebars from 'handlebars'
+import { customContentStyle } from './helpers/customContentStyle'
+import { links } from './helpers/links'
+
+handlebars.registerHelper('links', links)
+
+export default {
+  subject: `Your certification for {{{lockName}}}`,
+  html: `<h1>A NFT certification for "{{lockName}}" was airdropped to you!</h1>
+<p><a href="{{certificationUrl}}">Here</a> is your NFT certification (#{{keyId}}) for <strong>{{lockName}}</strong> was just airdropped!</p>
```

Comment:
Eventually we can make these as PDF that are attached to the email

Reply:
```suggestion
<p><a href="{{certificationUrl}}">Here</a> is your NFT certification (#{{keyId}}) for <strong>{{lockName}}</strong>. It was also just airdropped to your address.</p>
```

If the certification was airdropped by email we need to show the link to claim... please add that!